qulice-checkstyle/src/test/java/com/qulice/checkstyle/CheckstyleValidatorTest.java

```diff
@@ -679,16 +679,17 @@ public void allowsUppercaseAbbreviationExceptions() throws Exception {
      * @throws Exception If error
      */
     @Test
+    @SuppressWarnings("unchecked")
```

Comment:
@dskalenko OK, I see other tests also have it, but what is the line causing the unchecked warning? Is it ` Matchers.hasItems(`?